architecture-compose/README.md

```diff
@@ -1 +1,96 @@
-# Architecture-compose
\ No newline at end of file
+## Architecture-compose
+This Android library contains composable functions to work with Kaluga states and androidx navigation.
+
+## Installing
+This library is available on Maven Central. You can import Kaluga Base as follows:
+
+```kotlin
+repositories {
+    // ...
+    mavenCentral()
+}
+// ...
+dependencies {
+    // ...
+    implementation("com.splendo.kaluga.architecture.compose:$kalugaVersion")
+}
+```
+
+## Usage
+###Examples
+
+Inject view model with Koin and link it to lifecycle callbacks and a hardware back button
```

Comment:
I think readme example is better to be showed without other dependencies like koin. Then you can implement it with koin on kaluga example project

Reply:
but the lib contains a helper [method](https://github.com/splendo/kaluga/pull/379/files/36de730dc76cee2d6241e94a65008218bb9a1cfb#diff-a9c5b1f6d30fc1e08c880fe407897f03523a86f37a2c5714f798d06d8e26cdf8R21-R26) which was created for this purpose

Reply:
Right, saw that